scripts/boto3/multithread_download/uh_download.py

```diff
@@ -106,5 +110,11 @@ def set_total(self, total):
     end = time.monotonic()
     seconds = end - start
     mb = size_total / (1024 * 1024)
+    
+    if (config.otel_url):
+        otel = otel_exporter(config.otel_url[0], config.test_name[0])
+        otel.create_metric("bandwidth")
```

Comment:
please call the metric "download_bandwidth", otherwise the "bandwidth" metrics of upload and download tests will get mashed up or at least will be more cumbersome to distinguish.